### DISCUSSION TITLE
Multiplexing Sequelize model queries

### DISCUSSION THREAD

**Discussion by sravand123** [sequelize/sequelize#15434] [Help & Questions] [answered]
I was wondering if there is a way to  run multiple model queries  in a single database call,
For eg. 
`await Sequelize.run( User.findAll(...) , Post.findAll(...)).then(users, posts) { 
   ....
}` 
I can use Promise. all, but this will make multiple network calls to the database.

**Comment by ephys** [comment, marked as answer]
Such a thing is not possible as far as I know (do let us know if you have resources that point to the contrary). While you can put multiple queries in the same request, you'd only get the output of the last one
